lib/gui/pages/finish/templates/success.tpl.html

```diff
@@ -15,20 +15,20 @@ <h3 class="title"><span class="tick tick--success space-right-tiny"></span> Flas
     <div class="box center">
       <div class="fallback-banner">
         <div class="caption caption-big">Thanks for using
-          <svg-icon path="../../../assets/etcher.svg"
-            width="150px"
-            height="auto"
+          <svg-icon path="'../assets/etcher.svg'"
```

Comment:
> Actually it doesn't affect this module luckily

Huh? Surely svg-icon will still fail if the paths to the SVGs in the templates are incorrect? Which is an error we should attempt to detect?